OZON.ru.js

```diff
@@ -9,596 +9,130 @@
 	"inRepository": true,
 	"translatorType": 4,
 	"browserSupport": "gcsibv",
-	"lastUpdated": "2014-04-03 18:47:00"
+	"lastUpdated": "2015-09-20 18:13:00"
 }
 
-/**
-	OZON.ru translator. Only supports "Book" "Item Type".
-	Copyright (c) 2012 Victor Rybynok, v.rybynok@gmail.com
 
-	Description
-	===========
+function detectWeb(document, url) {
+	var URLRegexp = /https?:\/\/www\.ozon\.ru\/context\/detail\/id\/\d+\//;
+    var isURLValid = URLRegexp.test(url);
+    var hasISBN = xpath('//*[@itemprop="isbn"]/text()', document).iterateNext() !== null;
```

Comment:
Use `ZU.xpathText(doc, '//*[@itemprop="isbn"]`)`, which returns `null` if there is no xpath match